doc/sphinx-guides/source/developers/big-data-support.rst

```diff
@@ -16,7 +16,7 @@ Data Capture Module (DCM) is an experimental component that allows users to uplo
 Install a DCM
 ~~~~~~~~~~~~~
 
-Installation instructions can be found at https://github.com/sbgrid/data-capture-module . Note that a shared filesystem (posix or AWS S3) between Dataverse and your DCM is required. You cannot use a DCM with Swift at this point in time.
+Installation instructions can be found at https://github.com/sbgrid/data-capture-module/blob/master/doc/installation.md. Note that shared storage (posix or AWS S3) between Dataverse and your DCM is required. You cannot use a DCM with Swift at this point in time.
```

Comment:
As part of this effort, should we move "big data support" from the dev guide to some other guide?

Reply:
That seems like it's out of scope for this one.

Reply:
Ok, thanks.